### ISSUE TITLE
Say all pauses in the wrong places

### ISSUE THREAD

**Issue by tspivey** [nvaccess/nvda#13829]
### Steps to reproduce:
1. Open wordpad (notepad also works, but breaks slightly differently).
2. Paste this in:
```
Mr. and Mrs. Dursley, of number four, Privet Drive, were proud to say that they were perfectly normal, thank you very much.
```
3. Switch to OneCore, press Control+Home, then start a say all.
### Actual behavior:
NVDA pauses at Mr and/or Mrs, depending on which editor you use. OneCore is perfectly capable of expanding these, as demonstrated by reading the line.

### Expected behavior:
The reading shouldn't pause. NVDA should send larger chunks to the synthesizer.

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2022.1
#### Windows version:
Win10 X64 Version 21H1 (OS Build 19043.1766)
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by cary-rowen** [comment]
Hi,
I have not reproduced this in Notepad using the One Core speech synthesizer.
Is this related to One Core's language? I am using Simplified Chinese.

NVDA2022.2Beta2

**Comment by tspivey** [comment]
I tested with OneCore US English.

**Comment by CyrilleB79** [comment]
On my side, the issue is confirmed with OneCore and eSpeak English.
The issue is confirmed with WordPad and MS Word 2016.

But on the contrary to @tspivey, I cannot confirm the issue in Notepad (Windows 10 20H2 (x64) build 19042.1706).

**Comment by tspivey** [comment]
There was a newline after "normal,", which I removed to make it a single line. In wordpad that doesn't matter in this example, but that fixes it for notepad. Putting that back makes it pause after Mrs.

**Comment by CyrilleB79** [comment]
Yes, I confirme that, with the newline after "normal" in Notepad and in sayAll, "Mr." is pronounced correctly but "Mrs." is pronounced literally (i.e. "M r s") and there is an extra pause due to the dot character.

**Comment by michaelDCurran** [comment]
This seems to be Onecore choosing to do this specifically.
Generally Onecore will do a big pause after any words followed by a full stop. However, Mr. and Mrs. are specifically handled by Onecore to not pause because they would appear in the middle of a sentence.

If we take the above example, set symbol level to None, but add the word "dot" before the full stops:
```
Mr dot. and Mrs dot. Dursley, of number four, Privet Drive, were proud to say that they were perfectly 
```

We here the pause.

**Comment by tspivey** [comment]
This doesn't need an external fix, this is a bug in NVDA.
The expected behaviour is for NVDA to send enough to the synthesizer so it can process text without pausing in unexpected places. In US English, the period is either the end of the sentence or an abbreviation, and you don't really know which.
In that case, NVDA should be sending paragraphs to the synthesizer where it can.

**Comment by CyrilleB79** [comment]
I fully agree with @tspivey.
OneCore is able to differentiate the dot of an abbreviation from the dot being a full stop at the end of a sentence, provided it is given a large enough chunk of text. It's NVDA's responsibility to provide a chunk of text large enough.

@seanbudd could you please consider removing the blocked/needs-external-fix label? Thanks.

**Comment by michaelDCurran** [comment]
Yeah sorry,  this would definitely be an issue with sayAll. I've removed the needsExternalFix label now. What I pointed out before was separate to the bug you are talking about.

**Comment by LeonarddeR** [comment]
I wonder whether #20467 can somehow help with this, either it already does or paragraph should be added as an option.
Especially for non-braille users, throwing paragraphs through the say all pipeline is fine I think.
